contracts/contracts/PoolDeposits.sol

```diff
@@ -273,6 +283,30 @@ contract PoolDeposits {
     emit ProposalWithdrawn(msg.sender);
   }
 
+  function _distribute(
+    address[] calldata receivers,
+    uint256[] calldata percentages,
+    address winner,
+    uint256 iteration,
+    uint256 totalInterestFromIteration,
+    address tokenContract
+  ) internal {
+    IERC20 payoutToken = IERC20(tokenContract);
+
+    uint256 winnerPayout = totalInterestFromIteration;
```

Comment:
We now subtract from this, instead of subtracting from 100%.

This is more gas efficient (mul and div don't need to be performed to calculate winning), and there is no risk of rounding errors, it was never a problem because it always rounds down - but this is more precise - we always send the full amount of interest available, regardless of division errors.

Reply:
Cool got it